src/components/ChatLog.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+
+
+const ChatLog = (props) => {
+
+    const chats = props.entries.map((entries) => {
+        return (
+            < ChatEntry
+                toggleLike={props.toggleLike}
+                key={entries.id}
+                id={entries.id}
+                sender={entries.sender}
+                body={entries.body}
+                timeStamp={entries.timeStamp}
+                liked={entries.liked}
+            />
+        )
+    })
+    return <div>{chats}</div>;
+}
+
```

Comment:
Remember to put your `props-types` checking for this one. You could use the `arrayOf` and then `shape` functionality to be very explicit about the data structure you are expecting to pass through here.